components/Tile/src/Tile.tsx

```diff
@@ -9,17 +9,34 @@ const tileSize = {
     xlarge: 200,
 };
 
+/**
+ * Resolve a color variable or value for tile background.
+ * - If `color` is a palette path, returns the value from theme.palette
+ * - If `color` is a literal (hex/rgba/currentColor), returns it as-is
+ * - If `color` is undefined, falls back to the default theme surface color
+ */
+const resolveTileColor = (color?: IconColorProp): string => {
+    if (!color) return themeVars.layout.surfaceMid;
+
+    // Literal colors and currentColor pass through
+    if (color === 'currentColor') return color;
+    if (typeof color === 'string' && (color.startsWith('#') || color.startsWith('rgba('))) return color;
+
+    // Otherwise treat as a theme palette path
+    return cssVar(`palette.${String(color)}`);
+};
+
 const sizeToUnits = (size?: number | string) => {
     if (!size) return undefined;
     return typeof size === 'number' ? `${size}px` : size;
 };
 
-const StyledTileDiv = styled.div`
+const StyledTileDiv = styled.div<{ $color?: IconColorProp }>`
     display: flex;
     align-items: center;
     justify-content: center;
 
-    background-color: ${themeVars.layout.surfaceMid};
+    background-color: ${({ $color }) => resolveTileColor($color)};
```

Comment:
keep css variables here in styled. assign dynamic style via styles in the component